<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="6" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-30T22:39:47 IST" name="Suite" finished-at="2023-05-30T22:40:06 IST" duration-ms="19055">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidationTest.LoginErrorValidation()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationTest@68dc098b]" name="LoginErrorValidation" class="rahulshettyacademy.tests.ErrorValidationTest"/>
      </group> <!-- ErrorHandling -->
      <group name="Purchase">
        <method signature="submitOrderTest.submitOrder(java.util.HashMap)[pri:0, instance:rahulshettyacademy.tests.submitOrderTest@1b8a29df]" name="submitOrder" class="rahulshettyacademy.tests.submitOrderTest"/>
      </group> <!-- Purchase -->
    </groups>
    <test started-at="2023-05-30T22:39:47 IST" name="Error Validation Test" finished-at="2023-05-30T22:40:04 IST" duration-ms="16340">
      <class name="rahulshettyacademy.tests.ErrorValidationTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationTest@68dc098b]" started-at="2023-05-30T22:39:47 IST" name="launchApplication" finished-at="2023-05-30T22:39:54 IST" duration-ms="6697" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="LoginErrorValidation()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationTest@68dc098b]" started-at="2023-05-30T22:39:54 IST" name="LoginErrorValidation" finished-at="2023-05-30T22:39:55 IST" duration-ms="553" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email  password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email  password.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at rahulshettyacademy.tests.ErrorValidationTest.LoginErrorValidation(ErrorValidationTest.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDowm()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationTest@68dc098b]" started-at="2023-05-30T22:39:55 IST" name="tearDowm" finished-at="2023-05-30T22:39:55 IST" duration-ms="184" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDowm -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationTest@68dc098b]" started-at="2023-05-30T22:39:55 IST" name="launchApplication" finished-at="2023-05-30T22:39:57 IST" duration-ms="1771" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="LoginErrorValidation()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationTest@68dc098b]" started-at="2023-05-30T22:39:57 IST" name="LoginErrorValidation" finished-at="2023-05-30T22:39:57 IST" duration-ms="467" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email  password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email  password.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at rahulshettyacademy.tests.ErrorValidationTest.LoginErrorValidation(ErrorValidationTest.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDowm()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationTest@68dc098b]" started-at="2023-05-30T22:39:57 IST" name="tearDowm" finished-at="2023-05-30T22:39:57 IST" duration-ms="59" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDowm -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationTest@68dc098b]" started-at="2023-05-30T22:39:57 IST" name="launchApplication" finished-at="2023-05-30T22:40:01 IST" duration-ms="3206" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="ProductErrorValidation()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationTest@68dc098b]" started-at="2023-05-30T22:40:01 IST" name="ProductErrorValidation" finished-at="2023-05-30T22:40:04 IST" duration-ms="2921" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method is-config="true" signature="tearDowm()[pri:0, instance:rahulshettyacademy.tests.ErrorValidationTest@68dc098b]" started-at="2023-05-30T22:40:04 IST" name="tearDowm" finished-at="2023-05-30T22:40:04 IST" duration-ms="109" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDowm -->
      </class> <!-- rahulshettyacademy.tests.ErrorValidationTest -->
    </test> <!-- Error Validation Test -->
    <test started-at="2023-05-30T22:39:47 IST" name="submit Order Test" finished-at="2023-05-30T22:40:06 IST" duration-ms="19055">
      <class name="rahulshettyacademy.tests.submitOrderTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshettyacademy.tests.submitOrderTest@1b8a29df]" started-at="2023-05-30T22:39:48 IST" name="launchApplication" finished-at="2023-05-30T22:39:54 IST" duration-ms="5997" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:rahulshettyacademy.tests.submitOrderTest@1b8a29df]" started-at="2023-05-30T22:39:54 IST" name="submitOrder" data-provider="getData" finished-at="2023-05-30T22:39:58 IST" duration-ms="4627" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Pass@1234, product=ZARA COAT 3, email=xyzz@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDowm()[pri:0, instance:rahulshettyacademy.tests.submitOrderTest@1b8a29df]" started-at="2023-05-30T22:39:58 IST" name="tearDowm" finished-at="2023-05-30T22:39:58 IST" duration-ms="64" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDowm -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshettyacademy.tests.submitOrderTest@1b8a29df]" started-at="2023-05-30T22:39:58 IST" name="launchApplication" finished-at="2023-05-30T22:40:00 IST" duration-ms="1719" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:rahulshettyacademy.tests.submitOrderTest@1b8a29df]" started-at="2023-05-30T22:40:00 IST" name="submitOrder" data-provider="getData" finished-at="2023-05-30T22:40:04 IST" duration-ms="3972" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Pass@1234, product=ADIDAS ORIGINAL, email=sagar112@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDowm()[pri:0, instance:rahulshettyacademy.tests.submitOrderTest@1b8a29df]" started-at="2023-05-30T22:40:04 IST" name="tearDowm" finished-at="2023-05-30T22:40:04 IST" duration-ms="87" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDowm -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshettyacademy.tests.submitOrderTest@1b8a29df]" started-at="2023-05-30T22:40:04 IST" name="launchApplication" finished-at="2023-05-30T22:40:06 IST" duration-ms="1546" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="OrderHistoryTest()[pri:0, instance:rahulshettyacademy.tests.submitOrderTest@1b8a29df]" started-at="2023-05-30T22:40:06 IST" name="OrderHistoryTest" finished-at="2023-05-30T22:40:06 IST" depends-on-methods="rahulshettyacademy.tests.submitOrderTest.submitOrder" duration-ms="606" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistoryTest -->
        <test-method is-config="true" signature="tearDowm()[pri:0, instance:rahulshettyacademy.tests.submitOrderTest@1b8a29df]" started-at="2023-05-30T22:40:06 IST" name="tearDowm" finished-at="2023-05-30T22:40:06 IST" duration-ms="44" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDowm -->
      </class> <!-- rahulshettyacademy.tests.submitOrderTest -->
    </test> <!-- submit Order Test -->
  </suite> <!-- Suite -->
</testng-results>
